e2e/cypress/integration/at_mentions/at_mentions_spec.js

```diff
@@ -224,4 +230,76 @@ describe('at-mention', () => {
             cy.get('#sidebarItem_saepe-5').click({force: true});
```

Comment:
When all tests of this spec file are run, the newly added test is failing due to above line.  Please delete.
```suggestion
```

Reply:
I fixed it